### PR TITLE
Fix granule tests

### PR DESCRIPTION
**Pull request by yjpa7145**
The updates made for CUMULUS-223 broke the tests in the message-updates branch.  Fixed those tests.

### CODE REVIEW

File: cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js [hunk 1]
```diff
@@ -1,22 +1,13 @@
 'use strict';
 
+const aws = require('@cumulus/common/aws');
 const test = require('ava');
-const proxyquire = require('proxyquire');
```
Comment (scisco) [inline cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js]: @yjpa7145 I think we can also remove this as a dependency in the package.json

File: packages/common/aws.js [hunk 2]
```diff
@@ -146,47 +146,15 @@ exports.downloadS3File = (s3Obj, filename) => {
...
-* @param {object} options
-* @param {object} options.body
-* @param {object} options.bucket name of bucket
-* @param {object} options.key key for object (filepath + filename)
-* @param {object} options.body body of the file
-* @param {string} [options.acl=private] body of the file
-* @param {object} [options.Metadata=null] body of the file
-* @returns {promise} returns response from `S3.putObject` as a promise
-**/
-exports.putS3Object = (options) => {
```
Comment (scisco) [inline packages/common/aws.js]: Didn't get a chance to check but are we sure this function is not used by any of the tasks or libraries?

File: cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js (hunk 1 above)
Comment (yjpa7145) [reply cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js]: Fixed in 84fee36.

File: packages/common/aws.js (hunk 2 above)
Comment (yjpa7145) [reply packages/common/aws.js]: It isn't used anywhere.

File: .eslintrc.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
+  "root": true,
```
Comment (abarciauskas-bgse) [inline .eslintrc.json]: I may be misunderstanding but I think `"root": true` is only necessary in nested folders, when the interest is to stop searching for eslintrc in parent directories and use the eslintrc in the root of the nested directory, reading https://eslint.org/docs/user-guide/configuring

Comment (yjpa7145) [reply .eslintrc.json]: That is true, but there's no guarantee that someone won't check out this repo into a directory with a .eslintrc file above it.

**Pull request merged**